src/bin/parser_test.rs

```diff
@@ -22,29 +19,7 @@ pub struct TestResults {
     failed_position: usize,
 }
 
-struct Test {
-    /// Filename of the test
-    file_path: String,
-    /// Line number of the test
-    line: usize,
-    /// input stream
-    data: String,
-    /// errors
-    errors: Vec<Error>,
-    /// document tree
-    document: Vec<String>,
-    /// fragment
-    document_fragment: Vec<String>,
-}
-
-fn main() -> io::Result<()> {
-    let default_dir = "tests/data/html5lib-tests".to_string();
-
-    if !Path::new(&default_dir).exists() {
-        println!("Missing 'html5lib-tests'. It is not located at '{default_dir}' \nIt should be cloned first from here https://github.com/html5lib/html5lib-tests");
-        exit(1)
-    }
```

Comment:
With this PR, we now drop the optional argument of specifying a fixture directory.